math/src/main/java/com/powsybl/math/graph/UndirectedGraphImpl.java

```diff
@@ -390,6 +472,18 @@ public void traverse(int v, Traverser traverser) {
         return paths;
     }
 
+    /**
+     * This method is called by {@link #findAllPaths(int, Function, Function, TIntArrayList, BitSet, List)} each time a vertex is traversed.
+     *
```

Comment:
The javadoc suggests that the name is not very well chosen :) Not a big issue though...

Reply:
Which name? The method name?

Reply:
Yes that's what I meant, because from the javadoc it does not sound like it finds all paths from somewhere, but after looking into what it does it's not so bad.
Maybe the javadoc could say "Recursively finds all paths starting from specified vertex of specified edge" or something like that, together with "is used by findAllPaths".

Not really important anyway to be honest :)